mk_mingw.mak

```diff
@@ -9,7 +9,7 @@ CFLAGS = -Wall -std=gnu99
 SIZE_T_FMT_CHAR='""'
 COMMON_DEFINES=-DUSE_SYSTEM_STRNLEN
 DEFINES = -DWIN32 $(REGEX_DEFINES) -DHAVE_PACKCC $(COMMON_DEFINES)
-INCLUDES = -I. -Imain -Ignu_regex -Ifnmatch -Iparsers
+INCLUDES = -I. -Ignu_regex -Ifnmatch -iquote parser -iquote main
```

Comment:
It this related to using pkg-config?
If not but the change improves something, I would like you to make a separate commit.